migrating_from_ocp_3_to_4/index.adoc

```diff
@@ -0,0 +1,86 @@
+
+[id="mtc-overview-advanced-migration-options"]
+== Advanced migration options
+You can automate your migrations and modify the `MigPlan` and `MigrationController` custom resources in order to perform large-scale migrations and to improve performance. You can check the following items:
+
+* xref:../migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc#migration-creating-registry-route-for-dim_advanced-migration-options-3-4[Create a registry route for direct image migration]
+* xref:../migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc#migration-configuring-proxies_advanced-migration-options-3-4[Configuring proxies]
+* xref:../migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc#migration-migrating-applications-api_advanced-migration-options-3-4[Migrating an application by using the MTC API]
+* xref:../migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc#migration-state-migration-cli_advanced-migration-options-3-4[Running a state migration]
+* xref:../migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc#migration-hooks_advanced-migration-options-3-4[Creating migration hooks]
+* xref:../migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc#migration-plan-options_advanced-migration-options-3-4[Editing, excluding, and mapping migrated resources]
+* xref:../migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc#migration-controller-options_advanced-migration-options-3-4[Configuring the migration controller for large migrations]
+
+[id="mtc-overview-troubleshooting-mtc"]
+== Troubleshooting migrations
+You can perform the following troubleshooting tasks:
+
+* xref:../migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc#migration-viewing-migration-plan-resources_troubleshooting-3-4[Viewing plan resources]
+* xref:../migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc#migration-viewing-migration-plan-log_troubleshooting-3-4[Viewing the migration plan aggregated log file]
+* xref:../migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc#migration-using-mig-log-reader_troubleshooting-3-4[Using the migration log reader]
+* xref:../migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc#migration-accessing-performance-metrics_troubleshooting-3-4[Accessing performance metrics]
+* xref:../migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc#migration-using-must-gather_troubleshooting-3-4[Using the `must-gather` tool]
+* xref:../migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc#migration-debugging-velero-resources_troubleshooting-3-4[Using the Velero CLI to debug Backup and Restore CRs]
+* xref:../migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc#migration-partial-failure-velero_troubleshooting-3-4[Debugging a partial migration failure]
```

Comment:
Remove debugging partial failure. It is a subsection of Velero CLI tool.

Reply:
acked